Algorithms.cpp

```diff
@@ -112,53 +137,74 @@ int bubbleSort(int arr[],int length)
         } else { 
             cout << "," << arr[i] ;
         }
+        
         if(i == length-1) cout << "]" << endl << endl;
     }
-    return 0;
```

Comment:
Only necessary in int main, I believe. Also, per offline conversation, functions which modify mutable data structures don't require returning value (estou abrendendo)